packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -60,22 +75,37 @@ export interface DesignSystem {
      */
     cornerRadius?: number;
 
-    /*
+    /**
      * The width of the outline in pixels applied to outline components
      */
     outlinePatternOutlineWidth?: number;
+
+    /**
+     * The width of the outline for focus state in pixels.
+     */
+    focusOutlineWidth: number;
+
+    /**
+     * The opacity to use for disabled component state.
+     */
+    disabledOpacity: number;
 }
 
 const designSystemDefaults: DesignSystem = {
     backgroundColor: white,
     contrast: 0,
-    density: 1,
+    density: 0,
     designUnit: 4,
+    defaultHeightMultiplier: 8,
+    defaultPaddingMultiplier: 3,

```

Comment:
Is this always padding? Similar to @janechu's question on the function being applied to more than padding

Reply:
Changed to "horizontalSpacing"